### PR TITLE
Prepare for release 0.8.0

### PR DESCRIPTION
**Pull request by scottpurdy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0
+
+* Fix HTMPredictionModel prediction using SDRClassifier (#3665)
+* Allow multiple classifications for each record to SDRClassifier (#3669)
+* Remove CLAClassifier (#3665)
+* Add capnp serialization to TMRegion (#3657)
+* Document ExperimentDescriptionAPI (#3679)
+
```
Comment (rhyolight) [inline CHANGELOG.md]: I would add this because it was a breaking change:

```
* Updated BacktrackingTMCPP compute parameter name (#3667)
```

Comment (scottpurdy) [inline CHANGELOG.md]: Interestingly, I don't see the merge commit for #3667 when I run this command:

```
git log --merges 0.7.0..HEAD
```

Are you using one of the other merge methods that doesn't create a merge commit?

Comment (rhyolight) [inline CHANGELOG.md]: Not that I'm aware of. I just pushed the merge button.

Comment (rhyolight) [inline CHANGELOG.md]: https://github.com/numenta/nupic/pull/3667/commits/96dcdace8a7babc0732eb88f72b2c21b6a1959c3

Comment (scottpurdy) [inline CHANGELOG.md]: I don't think that commit is in numenta/nupic master.

Comment (scottpurdy) [inline CHANGELOG.md]: I don't really understand but I can see f83fb8954 in my commit history so the change is definitely there, just not sure where 96dcdac is.

Comment (scottpurdy) [inline CHANGELOG.md]: updated changelog

**Pull request merged**